pyqt-apps/siriushla/li_di_scrns/main.py

```diff
@@ -0,0 +1,489 @@
+            widget.showUnits = True
+        elif widType == 'lineEdit':
+            widget = PyDMLineEdit(
+                init_channel=pvName)
+            widget.setAlignment(Qt.AlignCenter)
+            widget.showUnits = True
+        elif widType == 'motorBtn':
+            widget = MotorBtn(
+                init_channel=pvName)
+        else:
+            widget = QLabel("Error: Unknown device")
+        widget.setMinimumWidth(60)
+        widget.setFixedHeight(20)
+        return widget
+
+    def getRadioBtn(self, device):
+        ''' Get the one radio button '''
+        radiobutton = QRadioButton(device)
+        radiobutton.device = device
+        radiobutton.toggled.connect(self.radioBtnClick)
+        if device == DEVICES[0]:
+            radiobutton.setChecked(True)
+        return radiobutton
+
+    def setBasicInfo(self, device, label, pv_name):
+        ''' Build one basic information Component '''
+        bi_hlay = QHBoxLayout()
+        bi_hlay.addWidget(
+            QLabel(label),
+            alignment=Qt.AlignCenter)
+        if label in ['Limit Mode', 'Motor Code', 'Counter', 'Sigma', 'Center']:
+            widType = 'label'
+        elif label == "Coefficient":
+            widType = 'spinBox'
+        elif label == "Centroid Threshold":
+            widType = 'lineEdit'
+        else:
+            widType = 'led'
+        bi_hlay.addWidget(
+            self.setWidgetType(widType, device, pv_name))
+        return bi_hlay
+
+    def setRBVObj(self, device, pv_name, label, pv_prefix):
+        ''' Build formatted RBV Component'''
+        rbv_hlay = QHBoxLayout()
+        title = QLabel(label)
+        title.setMinimumWidth(75)
+        title.setAlignment(Qt.AlignCenter)
+        rbv_hlay.addWidget(title)
+        for item in range(1, -1, -1):
+            pvName = pv_prefix + pv_name[item]
+            if 'RBV' not in pvName:
+                if label in ['Gain', 'Exposure']:
+                    widType = 'lineEdit'
+                else:
+                    widType = 'spinBox'
+            else:
+                widType = 'label'
+            widget = self.setWidgetType(widType, device, pvName)
+            widget.setMaximumWidth(75)
+            if item != 0:
+                rbv_hlay.addWidget(
+                        widget,
+                        alignment=Qt.AlignRight)
+                if 'Size' in pv_name[0]:
+                    sepLabel = QLabel("X")
+                    sepLabel.setFixedWidth(10)
+                    rbv_hlay.addWidget(
+                        sepLabel,
+                        Qt.AlignCenter)
+            else:
+                rbv_hlay.addWidget(
+                        widget,
+                        alignment=Qt.AlignLeft)
+        return rbv_hlay
+
+    def setGraph(self, device, graph_data, title):
+        '''Build a graph widget'''
+
+        graph_plot = PyDMWaveformPlot(background="#ffffff")
+        graph_plot.setTitle(title)
```

Comment:
I think it is better to use pydm title property:
`graph_plot.title = title`
or
`graph_plot.setPlotTitle(title)`